src/main/java/uk/ac/ebi/eva/pipeline/parameters/JobParametersNames.java

```diff
@@ -136,4 +136,6 @@
     public static final String CONFIG_RESTARTABILITY_ALLOW = "config.restartability.allow";
 
     public static final String CONFIG_CHUNK_SIZE = "config.chunk.size";
+
+    public static final String AGGREGATED_MAPPING_FILE = "aggregated.mapping.file";
```

Comment:
Wouldn't it be more intuitive to name this in a similar fashion to the aggregation mode?

[This document](https://docs.google.com/document/d/1ZMhLC-Mmd_1ygaFsdIRbcubJ9t4goEFnaLswk71v2Nw/) suggests `input.file.aggregation.mapping`, but given how the rest of arguments look then `input.vcf.aggregation.mapping-path` could be better.